### PR TITLE
Updates in order to use newest nullsafe version of flutter_map_marker_popup

### PR DESCRIPTION
**Pull request by rorystephenson**
Hey again @bsneider ! I've made some big updates to the nullsafe-2 branch for flutter_map_marker_popup which, apart from adding nullsafety, adds many bug fixes and improvements like handling rotation nicely. These changes should make it work with that version!

We still need to wait for flutter_map to publish null safety obviously.

Also it looks like flutter_map_marker_cluster is not supporting rotation yet but that can be tackled in a later PR even.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -24,9 +25,5 @@ dependency_overrides:
       ref: issues/829-nullsafety
   flutter_map_marker_popup:
     git:
-      url: git://github.com/bsneider/flutter_map_marker_popup.git
-      ref: nullsafety
-  flutter_image:
```
Comment (bsneider) [inline pubspec.yaml:29]: is flutter_image not needed any more, or perhaps now it is null safe?

File: example/pubspec.yaml [hunk 2]
```diff
@@ -36,10 +36,7 @@ dependency_overrides:
     git:
       url: git://github.com/fleaflet/flutter_map.git
       ref: issues/829-nullsafety
-  flutter_map_marker_popup:
```
Comment (bsneider) [inline example/pubspec.yaml:39]: if removed from dev_overrides, does a nullsafe version need to be added to dependencies?

**Review by bsneider — COMMENTED** [review]
@rorystephenson
Awesome, a couple of questions around the pubspec.yaml files, but very cool!

File: pubspec.yaml (hunk 1 above)
Comment (rorystephenson) [reply pubspec.yaml:29]: It is null safe now :)

File: example/pubspec.yaml (hunk 2 above)
Comment (rorystephenson) [reply example/pubspec.yaml:39]: Hmm I didn't try but maybe I should not have removed this one. Maybe add it back and then when flutter_map finally releases null safe and then I release null safe flutter_map_marker_popup we can remove these anyway.

File: pubspec.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   latlong2: ^0.8.0
+  flutter_map_marker_popup: ^0.3.0
```
Comment (danielocampo2) [inline pubspec.yaml]: @rorystephenson
`flutter_map_marker_popup` can be updated to ^1.0.0 now as it is null-safe.
`flutter_map` can be updated to ^0.13.1

Thanks for all the fixes! Looking forward for the null-safety release of this plugin.

File: example/pubspec.yaml (hunk 2 above)
Comment (rorystephenson) [reply example/pubspec.yaml:39]: Done now that it's moved to the new version.

File: pubspec.yaml (hunk 3 above)
Comment (rorystephenson) [reply pubspec.yaml]: Updated!

**Comment by rorystephenson** [comment]
@bsneider I think this should be ready to go.

**Pull request merged**